npm-shrinkwrap.json

```diff
@@ -2419,6 +2419,33 @@
       "from": "isstream@>=0.1.2 <0.2.0",
       "resolved": "https://registry.npmjs.org/isstream/-/isstream-0.1.2.tgz"
     },
+    "istanbul": {
```

Comment:
`isparta`, `browserify-istanbul` and `karma-coverage` appear to be missing from the shrinkwrap.

As an aside, I the know the shrinkwrap is a bit of a PITA to maintain at the moment. We might want to look at using the  [Yarn](https://yarnpkg.com/) npm client at some point which handles this much better.